import-export-cli/cmd/mg/deleteAPI.go

```diff
@@ -0,0 +1,103 @@
+/*
+*  Copyright (c) WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 Inc. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+ */
+
+package mg
+
+import (
+	"encoding/base64"
+	"fmt"
+
+	"github.com/spf13/cobra"
+	mgImpl "github.com/wso2/product-apim-tooling/import-export-cli/impl/mg"
+	"github.com/wso2/product-apim-tooling/import-export-cli/utils"
+	"golang.org/x/crypto/ssh/terminal"
+)
+
+var (
+	deleteAPICmdAPIName    string
+	deleteAPICmdAPIVersion string
+	deleteAPICmdAPIVHost   string
+	deleteAPIUsername      string
+	deleteAPIPassword      string
+)
+
+const deleteAPICmdLiteral = "api"
+const deleteAPICmdShortDesc = "Delete an API in Microgateway"
+const deleteAPICmdLongDesc = `Delete an API by specifying name, version, host, username 
+and optionally vhost by specifying the flags (--name (-n), --version (-v), --host (-c), 
+--username (-u), and optionally --vhost (-t). Note: The password can be included 
+via the flag --password (-p) or entered at the prompt.`
```

Comment:
Shall we change these lines similar to the above?
That's because, currently since we only have the ability to "Delete an API", I think the descriptions and usages of "delete" and "delete API" should be similar in both places.

Reply:
Please refer to the comment [here](https://github.com/wso2/product-apim-tooling/pull/604#discussion_r576526013)

Reply:
Shall we change this similar to what we have done in "apictl k8s delete API" [1]?

[1] https://github.com/wso2/product-apim-tooling/blob/master/import-export-cli/cmd/k8s/deleteApi.go

Reply:
Updated this.